### PR TITLE
Link edx stage and prod enviroments to share git SHAs

### PR DESCRIPTION
**Pull request by macdiesel**
Pairs well with: https://github.com/edx-ops/gomatic-secure/pull/51

@doctoryes PTAL.

### CODE REVIEW

File: edxpipelines/constants.py [hunk 1]
```diff
@@ -13,6 +13,8 @@
 RUN_PLAY_JOB_NAME = "run_play_job"
 APPLY_MIGRATIONS_STAGE = 'apply_migrations'
 APPLY_MIGRATIONS_JOB = 'apply_migrations_job'
+MANUAL_VERIFICATION_STAGE_NAME = 'manual_verification'
+MANUAL_VERIFICATION_JOB_NAME = 'manual_verification_job'
```
Comment (doctoryes) [inline edxpipelines/constants.py]: Nit: Maybe it's worth adding a check somewhere that these names match the ones coming in from the YAML?

**Comment by doctoryes** [comment]
Though I'm tempted to thumb down to avoid the merge conflicts with my rollback work:
👍 

😈

File: edxpipelines/constants.py (hunk 1 above)
Comment (macdiesel) [inline edxpipelines/constants.py]: I was thinking earlier that perhaps the constants file should actually be yaml, that way we could feed it in with the other configuration files at runtime and share the variable names. Was too big for the scope of the ticket.

**Comment by macdiesel** [comment]
@doctoryes hopefully not too many conflicts. I tried to stay out of the pattern files as much as I could.

**Pull request merged**